payments-core/src/main/java/com/stripe/android/paymentsheet/PaymentSheetAddPaymentMethodFragment.kt

```diff
@@ -71,4 +72,12 @@ internal class PaymentSheetAddCardFragment(
             googlePayButton.isEnabled = !isProcessing
         }
     }
+
+    private fun updateSelection() {
+        // TODO(brnunes-stripe): Refactor to support other payment methods
+        (
+            childFragmentManager
+                .findFragmentById(R.id.payment_method_fragment_container) as? AddCardFragment<*>
```

Comment:
Is it an error case if the Fragment isn't found?

Reply:
Right now this fragment is always being added in the base class when it's created. This will change as I implement the actual selector.